libs/features/webapp-prompt/focus-animation.css

```diff
@@ -0,0 +1,62 @@
+.coach-indicator {
+  --ring-color: ;
+  --coach-indicator-ring-default-color: rgba(56,146,243);
+  --coach-indicator-ring-diameter: 1.25rem;
+	--coach-indicator-ring-border-size: 2px;
+	--coach-indicator-ring-inline-size: var(--coach-indicator-ring-diameter);
+	--coach-indicator-ring-block-size: var(--coach-indicator-ring-diameter);
+  --animation-duration: 3000ms;
+}
+
+@keyframes pulse {
+	0% {
+		transform: scale(0.8);
+		opacity: 0;
+	}
+
+	50% {
+		transform: scale(1.5);
+		opacity: 1;
+	}
+
+	100% {
+		transform: scale(2);
+		opacity: 0;
+	}
+}
+
+.coach-indicator .coach-indicator-ring {
+	display: block;
+	position: absolute;
+  top: 14%;
+  left: 13%;
```

Comment:
I can see the argument for using pixels. UNav uses `rem` for the most part with the parent element having `font-size: medium`.

I use percentages in most other places since it allows me to reason about positioning using fractions. This turned out a bit wonky so I had to position it using a bit of trial and error, but I kept it percentage based for the consistency's sake.